### PR TITLE
Prepare 5.0.1

### PR DESCRIPTION
**Pull request by vweevers**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ _**If you are upgrading:** please see [`UPGRADING.md`](UPGRADING.md)._
...
+## [5.0.1] - 2019-04-07
+
+### Added
+
+- Test Electron in Travis and AppVeyor ([#607](https://github.com/Level/leveldown/issues/607)) ([**@vweevers**](https://github.com/vweevers))
+
+### Removed
+
+- Remove `slump` devDependency ([`38ff274`](https://github.com/Level/leveldown/commit/38ff274)) ([**@vweevers**](https://github.com/vweevers))
+- Remove Electron prebuilds in favor of runtime-agnostic prebuilds ([#608](https://github.com/Level/leveldown/issues/608)) ([**@vweevers**](https://github.com/vweevers))
```
Comment (vweevers) [inline CHANGELOG.md]: Are we good with doing this in a patch release?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ _**If you are upgrading:** please see [`UPGRADING.md`](UPGRADING.md)._
...
+
+### Fixed
+
+- Temporarily skip `hallmark` test because it breaks CITGM ([#606](https://github.com/Level/leveldown/issues/606)) ([**@vweevers**](https://github.com/vweevers))
+
+**Historical Note** Although CITGM landed support for git repositories in response to v5.0.0 which fixes our use of git submodules and `hallmark`, we're keeping `hallmark` disabled because it is still experimental and technically out of scope for CITGM.
```
Comment (vweevers) [inline CHANGELOG.md]: OK?

Comment (ralphtheninja) [reply CHANGELOG.md]: Yep

File: CHANGELOG.md (hunk 1 above)
Comment (ralphtheninja) [reply CHANGELOG.md]: I think so. It fixes something and it's not a breaking change, right?

Comment (vweevers) [reply CHANGELOG.md]: It fixes Electron 4 but breaks Electron 3, that's where my doubt comes from

Comment (ralphtheninja) [reply CHANGELOG.md]: Fixing Electron 4 (which is the latest) trumps Electron 3 imo.

**Pull request merged**